client/ayon_core/hosts/houdini/plugins/create/create_alembic_camera.py

```diff
@@ -13,6 +13,7 @@ class CreateAlembicCamera(plugin.HoudiniCreator):
     label = "Camera (Abc)"
     family = "camera"
     icon = "camera"
+    staging_dir = "$HIP/ayon"
```

Comment:
I feel like that if we're making this customizable - we're just as well of making the FULL path for the filepaths customizable, with e.g. default template in settings being:

```
$HIP/`chs("subset")`/`chs("subset")`.{ext}
```

Maybe only excluding the extension or putting in `{ext}` as variable.

Reply:
> I feel like that if we're making this customizable - we're just as well of making the FULL path for the filepaths customizable

This was my feeling too.
But, I have no idea how to list the available keys because it can be confusing that it doesn't support the same keys as https://ayon.ynput.io/docs/addon_houdini_admin#update-houdini-vars-on-context-change

Reply:
>  we're just as well of making the FULL path for the filepaths customizable, with e.g. default template in settings being:

I made that in this commit https://github.com/ynput/ayon-core/pull/9/commits/28ace5313c3826f619fa2f65bae6c2d9ab28a501 but I'm not sure if I made it in a good way. 
please let me know what do you think.